src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
+const MAX_HEALTH = 100;
+const BITE_DAMAGE = 50;
+const DEAD_VALUE = 0;
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = MAX_HEALTH;
+    Animal.alive.push(this);
+  }
+
+  die() {
```

Comment:
The `die` method could be made more efficient. Currently, it uses `filter` which is fine, but ensure that you are not using it inside a loop where it might be called multiple times unnecessarily, which could lead to performance issues.